scripts/deploy.py

```diff
@@ -21,38 +23,72 @@ def cli():
 @network_option()
 @ape_cli_context()
 @click.option("--blueprint", default=None)
-def factory(cli_ctx, account, network, blueprint):
-    if not blueprint:
-        blueprint_bytecode = b"\xFE\x71\x00" + HexBytes(  # ERC5202 preamble
-            project.StreamManager.contract_type.deployment_bytecode.bytecode
-        )
-        # the length of the deployed code in bytes
-        len_bytes = len(blueprint_bytecode).to_bytes(2, "big")
-        blueprint = account.call(
-            networks.ecosystem.create_transaction(
-                # copy <blueprint_bytecode> to memory and `RETURN` it per EVM creation semantics
-                # PUSH2 <len> RETURNDATASIZE DUP2 PUSH1 10 RETURNDATASIZE CODECOPY RETURN
-                data=b"\x61"
-                + len_bytes
-                + b"\x3d\x81\x60\x0a\x3d\x39\xf3"
-                + blueprint_bytecode
+@click.option("--create2", default=None, help="A string tag for the create2 deployment salt")
+def factory(cli_ctx, account, network, blueprint, create2):
+    if create2:
+        # NOTE: This is the deployment address listed on the create2 deployer's github:
+        # https://github.com/pcaversaccio/create2deployer/tree/main#deployments-create2deployer
+        # TODO: Add SDK for create2 deployer
+        create2_deployer = Contract("0x13b0D85CcB8bf860b6b79AF3029fCA081AE9beF2")
```

Comment:
I wonder if deploying should be part of the SDK too so it is more testable

Reply:
My idea was that the `manager` address would be the transition point between the scripts and the SDKs. They all reference a specific deployment, those contracts weren't build for add-hoc deployments. Could look into it further down the line though, especially using the factory's `create` method properly